### PR TITLE
Newseed

### PR DESCRIPTION
**Pull request by Peartes**
Make the dynamic social link component working
Themed the newseed page

### CODE REVIEW

File: src/app.scss [hunk 1]
```diff
@@ -11,11 +11,11 @@
 
   > .logo {
     grid-area: logo;
-    background-color: $color-black;
+    // background-color: $color-black;
```
Comment (dkent600) [inline src/app.scss:14]: did you confirm that this commented-out code does not break anything?

would be better to just remove the lines than comment them out.

File: src/footer/footer.html [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
   <div class="footerContainer">
+    <hr />
     <div class="sections">
       <!-- <a href="#" click.delegate="contactUs()" target="_blank" rel="noopener noreferrer">Contact</a>
       <a href="https://docs.primedao.io/primedao/faqs" target="_blank" rel="noopener noreferrer">FAQ</a> -->
       <div class="section about">
         <div class="title">About Us</div>
         <div class="body">
-          Prime Launch is a platform for fair token launchesin a completely permissionless fashion powered by PrimeDAO.
+          Prime Launch is a platform for fair token launchesin a completely
```
Comment (dkent600) [inline src/footer/footer.html:10]: launchesin => launches in

File: src/footer/footer.html [hunk 3]
```diff
@@ -25,39 +29,69 @@
       <div class="section launch">
         <div class="title">Prime Launch</div>
         <div class="body">
-          <a href="#" target="_blank" rel="noopener noreferrer">Why Prime Launch?</a>
-          <a href="#" target="_blank" rel="noopener noreferrer">Upcoming Launches</a>
-          <a href="#" target="_blank" rel="noopener noreferrer">Host a Launch</a>
-          <a href="#" target="_blank" rel="noopener noreferrer">Prime Launch Docs</a>
+          <a href="#" target="_blank" rel="noopener noreferrer"
+            >Why Prime Launch?</a
```
Comment (dkent600) [inline src/footer/footer.html:33]: All of these kinds of changes are really unnecessary and in this case, are cluttering up the code.  They make it unnecessarily harder for reviewers to do their job.

Best practice is that we review our own PRs and remove unnecessary stuff like this, before inviting in reviewers...

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/app.scss (hunk 1 above)
Comment (Peartes) [reply src/app.scss:14]: I checked that this did not break anything the logo's background was set to black and I just remove that to use the containing divs background color but i could remove them too

File: src/footer/footer.html (hunk 2 above)
Comment (Peartes) [reply src/footer/footer.html:10]: Ha, I'll adjust this

Comment (Peartes) [reply src/footer/footer.html:10]: I'll adjust it anyways

File: src/styles/_colors.scss [hunk 4]
```diff
@@ -20,13 +20,15 @@ $color-orange: #ffc780;
...
+$color-darkpink-2: #403453;
+$color-darkpink-3: #1b1722;
 $color-pink: #c08eff;
 $button1-color: #64b0c8;
 $button2-color: $color-darkpink;
 $button3-color: $color-light-purple;
 $button4-color: $color-green;
 
-$global-background-color: $color-black;
+$global-background-color: $color-darkpink-3;
```
Comment (dkent600) [inline src/styles/_colors.scss:31]: really?  the background looks pretty black to me

File: src/styles/styles.scss [hunk 5]
```diff
@@ -10,6 +10,8 @@ html {
     color: $color-white;
     font-family: Aeonik;
     min-height: 100vh;
+    padding-left: 4rem;
```
Comment (dkent600) [inline src/styles/styles.scss:13]: I hope you are testing all the pages with a change like this.

This kind of change should not be in this PR.  But lets leave it this time, as long as you have tested.

File: src/styles/styles.scss [hunk 6]
```diff
@@ -88,15 +90,15 @@ html {
     .button4,
     .buttonBw {
```
Comment (dkent600) [inline src/styles/styles.scss:91]: should button5 be included here?

**Review by dkent600 — APPROVED** [review]

**Pull request merged**